definitions/contested/json/FixedLists/FixedLists.json

```diff
@@ -1663,5 +1663,35 @@
     "ID": "GeneralLetterAddressToType",
     "ListElementCode": "other",
     "ListElement": "Other"
+  },
+  {
+    "LiveFrom": "01/01/2017",
```

Comment:
can you not re-use the existing list that has the same values?

Reply:
I probably could, but does it not make more sense to have a unique entry / list for each collection in case they change in the future? Someone looking at this might be concerned that the value for Refusal Order is pointing to a General Letter fixed list